### DISCUSSION TITLE
Using fragments for things other than lists

### DISCUSSION THREAD

**Discussion by mil-ad** [marp-team/marp#302] [Q&A] [answered]
I was wondering if there's an easy way to use fragments for arbitrary <div>s and not just items in list?

For instance sometimes I want half of the slide content (containing multiple lists) to appear after the first half.

**Comment by yhatt** [comment, marked as answer]
- https://github.com/marp-team/marpit/issues/195#issuecomment-534819826
- https://github.com/marp-team/marpit/issues/209

**Reply by mil-ad** [reply]
Brilliant!
